vendor/engines/split_accounts/app/services/split_accounts/order_detail_splitter.rb

```diff
@@ -78,9 +80,9 @@ def apply_order_detail_remainders
     end
 
     def apply_reservation_remainders
-      return unless order_detail.reservation
-      applier = RemainderApplier.new(order_detail.reservation, split_order_details.map(&:reservation), splits)
-      applier.apply_remainders(reservation_attribute_splitter)
+      return unless order_detail.time_data.present?
```

Comment:
I think it's either to think of as either occupancy or reservation. I agree the name isn't great, though. I'm definitely open to suggestions, though at this point it's getting baked in pretty deep.